### DISCUSSION TITLE
PHP Syntax Highlighting issue

### DISCUSSION THREAD

**Discussion by fritzmg** [community/community#23241] [New to GitHub]
<p>A short while ago, something was changed with PHP syntax highlighting. Now it appears to be required to always write</p>
<pre><code>```php
&lt;?php
// …
```
</code></pre>
<p>i.e. including the PHP open tag - otherwise the following PHP code will not actually be syntax highlighted.</p>
<p>Why was this changed? It’s an unnecessary nuissance and semantically wrong in my opinion. Most of the time you are posting snippets and examples of PHP code, not fully fledged classes from beginning to end. Thus it is a bit weird if you simply show parts of your class, without any use statements, but having to include &lt;?php nonetheless. Also it is common (at least in the Symfony documentation for example) to not include the &lt;?php open tag in code examples.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/fritzmg">@fritzmg</a>,</p>
<p>Thanks for the feedback, we have an issue open in the linguist repository and are working on a fix. Apologies for the inconvenience. Here is that issue if you’d like to follow along:</p>
<p><a href="https://github.com/github/linguist/issues/4865" rel="nofollow noopener">https://github.com/github/linguist/issues/4865</a></p>

**Comment by fritzmg** [comment]
<p>Thank you :slight_smile:</p>
<p>I did try to search for any existing issues - however I find it hard to Google for any issues related to GitHub itself, since that would just show a lot of different GitHub repositories in the search result :grin:</p>
